Package.swift

```diff
@@ -78,6 +78,7 @@ let package = Package(
         .library(name: "FoundationPreview", targets: ["FoundationPreview"]),
         .library(name: "FoundationEssentials", targets: ["FoundationEssentials"]),
         .library(name: "FoundationInternationalization", targets: ["FoundationInternationalization"]),
+        .library(name: "FoundationBenchmarkSupport", targets: ["FoundationBenchmarkSupport"]),
```

Comment:
We shouldn't export this since it's for testing only.

Reply:
Moved this to the benchmark package as you suggested below